blaseball_mike/chronicler.py

```diff
@@ -133,3 +133,23 @@ def get_tribute_updates(before=None, after=None, order=None, count=None):
         params["count"] = count
 
     return paged_get(f'{BASE_URL}/tributes/hourly', params=params, session=cached_session)
+
+def time_map(season=0, tournament=-1, day=0, include_nongame=False):
```

Comment:
1 index the defaults!

Reply:
that was supposed to be a question mark not a !, i am not that strong opinioned here

Reply:
I zero indexed them so that they would be passed to Chronicler as `season=-1, tournament=-1, day=-1` since this function will do the 1-index conversion itself, which will just return an empty list

Reply:
ok gotcha